prepare-image.sh

```diff
@@ -28,6 +28,10 @@ rpm -q subscription-manager && \
 # Pbr pulls in Git (30+ MiB), but actually only uses it in development context.
 rpm -q git-core && rpm -e --nodeps git-core || true
 
+# NOTE(elfosardo): we need to reinstall tzdata as the base CS9 container removes
+# its content, for more info see https://bugzilla.redhat.com/show_bug.cgi?id=2052861
+dnf reinstall -y tzdata
```

Comment:
Please move this before the `rpm` commands above because they invalidate the dependencies

Reply:
oh snap! yeah...